08LessonEight/homework/css/exercise-06-style.css

```diff
@@ -1 +1,33 @@
-/*Add CSS rules*/
\ No newline at end of file
+.left{
+  width:30%;
+  float: left;
+  font-family: sans-serif;
+}
+.middle{
+  width:30%;
+  float: left;
+  font-family: sans-serif;
+}
+.right{
+  width:30%;
+  float: left;
+  font-family: sans-serif;
+}
+h4{
+  font-weight: bold;
+}
+p{
+  font-weight: lighter;
+  font-size: 14px;
+}
+a{
+  color: black;
+  text-decoration: none;
+  font-weight: bold;
+  font-size: 10px;
+  border-right: 3px #8C001A solid;
+  border-top:1px #d3d3d3 solid;
+  border-left:1px #d3d3d3 solid;
+  border-bottom:1px #d3d3d3 solid;
```

Comment:
if there's only one border that's different, you could make a general border declaration and then put border-right below it, so border-right will override the default